Project title: Investigating shared and distinct mechanisms of inflammatory diseases 

About the project: While transcriptomics has seen considerable use in investigating inflammatory diseases, most studies of gene expression consider only a single disease in isolation. Relatively few studies have attempted to investigate shared signatures of inflammatory diseases, in part due to the challenges involved in integrating expression across different experiments. We gained access to data from the IMID-Bio-UK network (https://www.gla.ac.uk/research/az/imid/aboutus/) and curated public datasets. This study will investigate the transcriptomes of inflammatory disease both in bulk RNA-seq and single-cell data. Signatures of disease and drug response will be identified by applying dimensionality reduction methods, such as weighted gene correlation network analysis (WGCNA) [1] and independent component analysis (ICA) [2]. The resulting signatures will then be compared across multiple datasets to differentiate between shared and distinct mechanisms of inflammatory disorders that might be targeted by new or repositioned therapies. Machine learning approaches could also be applied to compare and integrate these data; for instance, we could alternatively apply autoencoders to identify latent variables and compare diseases.

 

Plan 

Core project: comparing signatures of IMIDs 

    Data ingestion and exploration of IMID data (likely looking at Atopic Dermatitis in addition to existing Psoriasis and IBD datasets) 

    Comparing gene-level signatures of disease 

    Using dimensionality reduction to simplify and interpret IMID data 

    Applying dimensionality reduction across datasets to compare IMID signatures 



 

